rows.go

```diff
@@ -64,12 +65,24 @@ func (rows *mysqlRows) Columns() []string {
 	return columns
 }
 
+func (rows *mysqlRows) setFinish(f func()) {
```

Comment:
Here is another solution.

https://github.com/shogo82148/mysql/commit/9b979f24edb30940714b69b84e14e39b394691f6

Which do you like?